utility/output_writer_test.go

```diff
@@ -11,7 +11,7 @@ func TestWriteCustomOutput(t *testing.T) {
 }
 
 // TestExampleFWriteCustomOutput shows how to use the WriteCustomOutput function
-func TestExampleFWriteCustomOutput() {
+func TestExampleFWriteCustomOutput(t *testing.T) {
```

Comment:
The func name must start with `Example` and not with `Test` if you want to use the output. You don't need the `t *testing.T` argument.